src/interface/storage.thrift

```diff
@@ -557,24 +557,29 @@ struct LookupAndTraverseRequest {
  * End of Index section
  */
 
+struct ScanCursor {
+    3: bool                                 has_next,
```

Comment:
The `has_next` seems useless, just judge by `next_cursor` is enough?

Reply:
Yes, I just keep the origin fields.